core/rio/ntriples/src/main/java/org/eclipse/rdf4j/rio/ntriples/NTriplesParser.java

```diff
@@ -240,6 +247,17 @@ protected int skipLine(int c) throws IOException {
 		return c;
 	}
 
+	protected int skipLine(int c) throws IOException {
+		return skipLine(c, null);
+	}
+
+	private int parseComment(int c) throws IOException {
+		StringBuilder sb = new StringBuilder(100);
+		int res = skipLine(c, sb);
+		rdfHandler.handleComment(sb.toString());
```

Comment:
`rdfHandler` may be null (e.g. when the parser is used for verification only) so you need a null-check here.